lib/route-on-event.js

```diff
@@ -8,30 +8,32 @@ exports.routeFromLocation = function routeFromLocation (event, router) {
 }
 
 exports.routeFromLinkClick = function routeFromLinkClick (event, router) {
-  // ignore if it could open a new window, if a right click
-  if (
-    event.metaKey || event.shiftKey || event.ctrlKey || event.altKey ||
-    event.which === 3 || event.button === 2
-  ) return
+  setTimeout(() => {
```

Comment:
My bad.

On Thu, Jan 14, 2016 at 12:43 PM, Andy VanWagoner <notifications@github.com>
wrote:

> In lib/route-on-event.js
> <https://github.com/thetalecrafter/middle-router/pull/8#discussion_r49774720>
> :
>
> > @@ -8,30 +8,32 @@ exports.routeFromLocation = function routeFromLocation (event, router) {
> >  }
> >
> >  exports.routeFromLinkClick = function routeFromLinkClick (event, router) {
> > -  // ignore if it could open a new window, if a right click
> > -  if (
> > -    event.metaKey || event.shiftKey || event.ctrlKey || event.altKey ||
> > -    event.which === 3 || event.button === 2
> > -  ) return
> > +  setTimeout(() => {
>
> arrow functions are not supported in all supported browsers, and this code
> is not transformed using babel or any other tool.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thetalecrafter/middle-router/pull/8/files#r49774720>.
>